babel.config.js

```diff
@@ -23,4 +23,7 @@ module.exports = {
       },
     ],
   ],
+  ignore: [
+    "**/*.test.js",
+  ],
```

Comment:
is this ignore redundant with the one in package.json's transpileES command line?

Reply:
That line was seemingly not working for me. Removing single quotes around the path got it working again, as with `src`.